src/main/java/edition/academy/seventh/repository/BookParser.java

```diff
@@ -0,0 +1,42 @@
+package edition.academy.seventh.repository;
+
+import edition.academy.seventh.database.model.DTBook;
+import edition.academy.seventh.model.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Provides parsing {@link DTBook} into {@link BookstoreBook} and {@code List<BookstoreBook>} into
+ * {@code List<DTBook>}.
+ *
+ * @author Agnieszka Trzewik
+ */
+class BookParser {
+
```

Comment:
Należało by się zastanowić czy metody statyczne czy jednak tworzenie instancji / beana i metody obiektu.

Reply:
Po przemyśłeniu, przeczytaniu into na temat stosowania satycznych metod, np tutaj: https://stackoverflow.com/questions/11993077/difference-between-static-methods-and-instance-methods metody te odpowiadają idealnie istocie działania metod statycznych, jako metody pomocnicze dla BookRepository